docs/fieldnorm/Software life cycle processes.md

```diff
@@ -0,0 +1,67 @@
+# 1. Software life cycle processes
+
+## 1.1 General
+One of the main responsibilities of bioinformaticians within a diagnostic setting involves the development and maintenance of software used in diagnostic processes. Because ISO-15189 does not cover software development, it is essential that in-house software is created and maintained in accordance with recognized harmonized standards, such as IEC-62304. The IEC-62304 description of the software development life cycle (SDLC) is consistent with the practices of most software-centric entities. 
```

Comment:
Maybe we should start a separate introductory document for all these parts where everyone mentions that the iso-15189 is not suited for bioinformaticians. I would leave the part about the IEC-62304 description of the SDLC here because that does look specific to life cycle processes, but maybe merge it into 1.2

Reply:
I Removed the part about ISO15189 and merged 1.1 + 1.2 together